doc/source/theory/pQCD.rst

```diff
@@ -80,23 +85,83 @@ The expression of the pure |QED| and of the mixed |QED| :math:`\otimes` |QCD| sp
 Scale Variations
 ----------------
 
-The usual procedure in solving |DGLAP| that is also applied :doc:`here
+The usual procedure in solving |DGLAP| applied :doc:`here
 </theory/DGLAP>` is to rewrite the equations in term of the running coupling
 :math:`a_s` assuming the factorization scale :math:`\mu_F^2` (the inherit scale
 of the |PDF|) and the renormalization scale :math:`\mu_R^2` (the inherit scale
-for the strong coupling) to be equal. This constraint can however be lifted by a
-suitable redefinition of the splitting kernels :cite:`Vogt:2004ns`:
+for the strong coupling) to be equal.
+This constraint, however, can be lifted in order to provide an estimation of the
+missing higher order uncertainties (|MHOU|) coming from |DGLAP| evolution :cite:`AbdulKhalek:2019ihb`.
+Since scale-dependent contributions to a perturbative prediction are fixed by |RGE| invariance,
+the scale variation can be used to generate higher order contributions,
+which are then taken as a proxy for the whole missing higher orders.
+This method provides many advantages:
 
-.. math ::
-    \gamma^{(1)}(N) &\to \gamma^{(1)}(N) - \beta_0 \ln(\mu_F^2/\mu_R^2) \gamma^{(0)} \\
-    \gamma^{(2)}(N) &\to \gamma^{(2)}(N) - 2 \beta_0 \ln(\mu_F^2/\mu_R^2) \gamma^{(1)} - ( \beta_1 \ln(\mu_F^2/\mu_R^2) - \beta_0^2 \ln^2(\mu_F^2/\mu_R^2) )  \gamma^{(0)}
+    * it naturally incorporates renormalization group invariance,
+      as the perturbative order increases, estimates of |MHOU| decrease;
+    * the same procedure can be used for any perturbative process,
+      since the scale dependence of the strong coupling :math:`a_s(\mu^2)` and of PDFs is universal;
+
+However there is no unique principle to determine the specific range of the scale variation;
```

Comment:
- "However," (comma)
- I'd suggest: "principle" -> "prescription" ...
- also maybe avoid the semi-colon (if you have an easy fix)